### PR TITLE
General updates

### PR DESCRIPTION
**Pull request by Pilargit12**
Research on dead links and small updates, but through this fixed errors in navigation

### CODE REVIEW

File: src/docs/automation-and-resiliency/openshift-project-resource-quotas.md [hunk 1]
```diff
@@ -26,12 +27,13 @@ New project sets provisioned in **all clusters** of the BC Gov Private Cloud Paa
...
 
+If the default allocations aren't sufficient for your application, [you can ask for a quota increase](/request-quota-increase-for-openshift-project-set/). You'll need a Sysdig dashboard that shows that your application needs more of a specific resource type (CPU, RAM or storage) in a specific namespace. Provide this proof to the Platform Services team before they can approve a quota increase that you submit in the [Platform Project Registry](https://registry.developer.gov.bc.ca/public-landing).
+
 ## On this page
 - [CPU quotas](#cpu-quotas)
 - [Memory quotas](#memory-quotas)
 - [Storage quotas](#storage-quotas)
-
-If the default allocations aren't sufficient for your application, [you can ask for a quota increase](/request-quota-increase-for-openshift-project-set/). You'll need a Sysdig dashboard that shows that your application needs more of a specific resource type (CPU, RAM or storage) in a specific namespace. Provide this proof to the Platform Services team before they can approve a quota increase that you submit in the [Platform Project Registry](https://registry.developer.gov.bc.ca/public-landing).
```
Comment (mtspn) [inline src/docs/automation-and-resiliency/openshift-project-resource-quotas.md:34]: Did you intend to remove these lines @Pilargit12 ?

Comment (Pilargit12) [reply src/docs/automation-and-resiliency/openshift-project-resource-quotas.md:34]: I moved those above the On this page section instead

File: src/docs/build-deploy-and-maintain-apps/maintain-an-application.md [hunk 2]
```diff
@@ -358,7 +358,7 @@ Once your application is running in OpenShift, you can use [Sysdig](https://digi
 
 1.  [Onboard to Sysdig](/sysdig-monitor-onboarding/) and setup the OpenShift project set access to monitor your applications
 2. [Create Sysdig monitoring dashboards](https://docs.sysdig.com/en/docs/sysdig-monitor/dashboards/) to gather important application metrics such as resource utilization and service Golden Signals
-3. Merely having a monitoring dashboard isn't sufficient since you won't be actively checking it throughout the day. Instead, it's advisable to establish alerts for monitoring metrics. This approach ensures that you receive notifications when potential issues arise, allowing you to address problems in their early stages. Follow the guide on [how to setup Sysdig alerts via Rocket.Chat](/app-monitoring/sysdig-monitor-create-alert-channels/)
+3. Merely having a monitoring dashboard isn't sufficient since you won't be actively checking it throughout the day. Instead, it's advisable to establish alerts for monitoring metrics. This approach ensures that you receive notifications when potential issues arise, allowing you to address problems in their early stages. Follow the guide on [how to setup Sysdig alerts via Rocket.Chat](/sysdig-monitor-create-alert-channels/)
```
Comment (mtspn) [inline src/docs/build-deploy-and-maintain-apps/maintain-an-application.md:361]: Don't think this link format will work? Also, the other link further down this page is broken using this same format.

**Review by mtspn — COMMENTED** [review]
Changes look mostly good, please just confirm if your intention is to remove the lines describing increasing a quota in openshift-project-resource-quotas.md

**Review by mtspn — APPROVED** [review]

**Pull request merged**